[DOCS] Users page (#1882)
elastic/security-docs: docs/getting-started/users-page.asciidoc

[[users-page]]
= Users page

The Users page provides a comprehensive overview of user data to help you understand authentication and user behavior within your environment. Key performance indicator (KPI) charts, data tables, and interactive widgets let you view specific data and drill down for deeper insights. Here's some of the information available to you:

[role="screenshot"]
image::images/users/users-page.png[]

*User KPI charts*

> KPI charts show the total number of users, successful and failed user authentications within the time range specified in the date picker. Data in the KPI charts is visualized through linear and bar graphs.

Reviewer: ```suggestion
KPI charts show the total number of users and successful and failed user authentications within the time range specified in the date picker. Data in the KPI charts is visualized through linear and bar graphs.
```

> KPI charts show the total number of users, successful and failed user authentications within the time range specified in the date picker. Data in the KPI charts is visualized through linear and bar graphs.

Reviewer 2: ```suggestion
KPI charts show the total number of users, successful and failed user authentications within the time range specified in the date picker. Data in the KPI charts is visualized using line and bar graphs.
```

[[users-page]]
= Users

> The Users page provides a comprehensive overview of metric user data to understand authentication and usage information within your environment. Key Performance Indicator (KPI) charts, data tables, and interactive widgets let you view specific data and drill down for deeper insights. Here's some of the information you can view:

Reviewer 2: ```suggestion
The Users page provides a comprehensive overview of user data to help you understand authentication and usage information within your environment. Key Performance Indicator (KPI) charts, data tables, and interactive widgets let you view specific data and drill down for deeper insights. Here's some of the information you can view:
```
I could be missing something here, but "metric" seemed off.

> The Users page provides a comprehensive overview of metric user data to understand authentication and usage information within your environment. Key Performance Indicator (KPI) charts, data tables, and interactive widgets let you view specific data and drill down for deeper insights. Here's some of the information you can view:

Reviewer 3: Just curious, is "usage information" user behavior?

> The Users page provides a comprehensive overview of metric user data to understand authentication and usage information within your environment. Key Performance Indicator (KPI) charts, data tables, and interactive widgets let you view specific data and drill down for deeper insights. Here's some of the information you can view:

Reviewer 3: ```suggestion
The Users page provides a comprehensive overview of metric user data to understand authentication and usage information within your environment. Key Performance Indicator (KPI) charts, data tables, and interactive widgets let you view specific data and drill down for deeper insights. Here's some of the information available to you:
```

Reviewer 3: Mainly making this rec to avoid using "view".

Reviewer: * +1 @nastasha-solomon's comment, and/or maybe shorten to "Here's some of the information available:"

* Does "Key Performance Indicator" need to be capitalized? AFAIK it's a general term.

[role="screenshot"]
image::images/users/users-page.png[]

> *User Key Performance Indicators (KPIs) charts*

Reviewer 3: ```suggestion
*User KPI charts*
```

> KPI charts show the total number of users, successful and failed user authentications, and unique IPs based on the time range specified in the date picker. Data in the KPI charts is visualized through linear and bar graphs.

Reviewer 4: We don't display `unique IPs` KPI on the user page. It is displayed on the Host page.

> TIP: Place your cursor inside a KPI chart to view the actions menu (*...*), where you can perform these actions: inspect, open in Lens, add to a new case, and attach to an existing case.

Reviewer 3: ```suggestion
TIP: Place your cursor inside a KPI chart to access the actions menu (*...*), where you can perform these actions: inspect, open in Lens, and add to a new or an existing case.
```

Reviewer: "Hover" instead of "place your cursor"?

[role="screenshot"]
image::images/users/chart-menu.png[]

*Data tables*

Beneath the KPI charts are data tables, which are useful for viewing and investigating specific types of data. Select the relevant tab to view the following details:

> * *All users*: A chronological list of unique user names, the time they were last active, and the associated domain.

Reviewer 2: ```suggestion
* *All users*: A chronological list of unique user names, when they were last active, and the associated domains.
```

> * *Authentications*: A chronological list of user authentication events and their associated details, such as the number of successes, failures, and the hostname of the last successful destination.

Reviewer 2: ```suggestion
* *Authentications*: A chronological list of user authentication events and associated details such as the number of successes and failures, and the hostname of the last successful destination.
```

* *Anomalies*: Unusual activity discovered by machine learning jobs that contain user data.

> * *Events*: Ingested events that contain the `user.name` field. You can stack by `event.action`, `event.dataset`, or `event.module` fields.

Reviewer 2: I could be wrong but I think it's "field" not "fields" since you can only stack by one OR the other. i.e. "You can stack by the 'event.action' field".

> * *External alerts*: Alerts received from external monitoring tools. You can stack by `event.module` and `event.category` fields.

Reviewer 2: ```suggestion
* *External alerts*: Alerts received from external monitoring tools. You can stack by the `event.module` or `event.category` field.
```

> The tables within the *Events* and *External alerts* tabs are interactive and provide several options that allow you to customize the view. For example, you can investigate events in Timeline, choose which fields to display, change the sort order of columns, and view alert details in the Alert details flyout. To learn more about what you can do with these tables, refer to <<alerts-ui-manage>>.

Reviewer 2: ```suggestion
The tables within the *Events* and *External alerts* tabs are interactive and provide several customization options. For example, you can investigate events in Timeline, choose which fields to display, change the sort order of columns, and view alert details in the Alert details flyout. To learn more about what you can do with these tables, refer to <<alerts-ui-manage>>.
```

> The tables within the *Events* and *External alerts* tabs are interactive and provide several options that allow you to customize the view. For example, you can investigate events in Timeline, choose which fields to display, change the sort order of columns, and view alert details in the Alert details flyout. To learn more about what you can do with these tables, refer to <<alerts-ui-manage>>.

Reviewer 3: ```suggestion
The tables within the *Events* and *External alerts* tabs are interactive and provide several options that allow you to customize the view. For example, you can investigate events in Timeline, choose which fields to display, change the order of columns, and view alert details in the Alert details flyout. To learn more about what you can do with these tables, refer to <<alerts-ui-manage>>.
```

Reviewer 3: Not vital to remove, but I think the idea of being able to rearrange columns is communicated whether you keep "sort".

[role="screenshot"]

> image::images/users/events-table.png[]

Reviewer: Could the legend/explanations be moved to body text, instead of part of the image? That's how we've handled these kinds of diagrams elsewhere in the docs, and it's really hard to read the text when it's part of the image.

(Commenting here since I can't comment directly on an image.)
